pkg/common/utils.go

```diff
@@ -70,8 +70,9 @@ func BytesToGbRoundDown(bytes int64) int64 {
 }
 
 func BytesToGbRoundUp(bytes int64) int64 {
-	re := bytes / (1024 * 1024 * 1024)
-	if (bytes % (1024 * 1024 * 1024)) != 0 {
+	// TEST: verify that rounding to 4GiB works correctly
+	re := bytes / (1024 * 1024 * 1024 * 4)
```

Comment:
`re` is now nr. of 4 GiB chunks in `bytes`. When you `return re` below, you should `return 4 * re` to return nr. of GiB chunks and not 4GiB chunks.

This took me some time :-D